README.md

```diff
@@ -0,0 +1,63 @@
+# IronSSH - End-to-end secure file transfer
+
+This project is a fork of the *openssh/openssh-portable* project. While most of the project is unchanged, specific additions have been made to create new executables that provide end-to-end security when transferring files to remote servers.
```

Comment:
Lets link to the openssh project home page or github page.  Hmmm.  Let's link to their home page: openssh.com.